README.md

```diff
@@ -1,8 +1,50 @@
 # ansible-dkim
 
-Ansible role for configuring [Postfix](http://www.postfix.org/) with [OpenDKIM](http://opendkim.org/) support on [Ubuntu](https://ubuntu.com/).
+Ansible role for configuring [Postfix](http://www.postfix.org/) with [OpenDKIM](http://opendkim.org/). Works on [Debian](https://debian.org) distributions and derived like [Ubuntu](https://ubuntu.com/).
 
-### Example playbook
+## Description
+
+This role confiures DKIM mail singing service in a hosts that works as e.ail MTA. 
+
+The role: 
+* installs and configures opendkim,
+* creates private and public dkim keys for the domains it has to sign, declared in the `dkim_domains` variable,
+* installs postfix and configures it to pass all the messages of the configured domains to be signed by OpenDKIM,
+* shows the DNS records with the public keys that must be defined in the public DNS of the domains we sign.
+
+## Requirements
+
+The role requires that you configure all the rest of the mail management and you will need to have access to the DNS configuration of the domains you are requiesting to sign. At the end, the role will give you the DNS records with the public keys of the domain that you will have to publish in the global DNS system.  
+
+## Role variable
+
+### Opendkim package parameters
+
+|  Variable     |   Default value   |   Description  |
+|:-------------------:|:------------------------:|:------------:|
+| `dkim_default_config_file:` | /etc/default/opendkim | Opendkim Default values configuration file |
+| `dkim_opendkim_config:` | /etc/opendkim | Opendkim configuration folder |
+| `dkim_user:` | opendkim | linux user that runs Opendkim | 
+| `dkim_group:` | opendkim | linux group that runs Opendkim | 
+
+### Opendkim configuration parameters
+
+|  Variable     |   Default value   |   Description  |
+|:-------------------:|:------------------------:|:------------:|
+| `dkim_selector:` | email | Opendkim registers' selector |
+| `dkim_admin_email:` | none | e-mail address that manages opendkim. You musrt define either `dkim_admin_email` or legacy `admin_email`. |
+| `dkim_domains:` | none | List of domains that opendkim must be configured to sign the mails of. A yaml list of DNS. |
+| `dkim_same_key:` | true | Whether opendkim must use the same keys for all domains or a set of keys for each domain.  |
```

Comment:
You can use `false` as default since it's more secure.

Reply:
What do you mean here by 'secure'? I put the default  value `true` to maintain the behaviour for you and all previous users of the role. If we set the default value to false, all of you will have to be aware that you have to define `dkim_same_key: true` in your playbooks if you want to preserve the use of one key for all declared domains. 
But I'll do as you wish.

Reply:
It's more secure to have separate key for each domain. That way if one key gets compromised, it won't affect other keys. But if you want to maintain backward compatibility first, no problem. I leave it up to you.

Reply:
I think it's better do for the good of your users, don't break their code.